doc/source/ray-air/examples/convert_existing_tf_code_to_ray_air.ipynb

```diff
@@ -0,0 +1,1070 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result.metrics_dataframe.plot(\"training_iteration\", [\"train_loss\", \"test_loss\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9754329",
+   "metadata": {},
+   "source": [
+    "### Continue training from a checkpoint\n",
+    "\n",
+    "In the case that your training run gets interrupted (e.g. intentionally interrupted by a user or a preempted by a spot instance), we may want to resume training from a checkpoint.\n",
+    "\n",
+    "We can do this by creating a new trainer and supplying a checkpoint (in this case, a `TensorflowCheckpoint`). In the example below, we continue training our model for 1 epoch, retrieving the checkpoint from the previous training result. Another method of retrieving the checkpoint is from a checkpoint directory path, and we given an example of what that looks like in the comments below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2b11ee03",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the last checkpoint from the previous training run\n",
+    "tf_checkpoint: TensorflowCheckpoint = result.checkpoint\n",
+    "\n",
+    "# Alternatively, load from a checkpoint directory, for example:\n",
+    "# checkpoint_path = \"~/ray_results/exp_name/TensorflowTrainer_.../checkpoint_...\"\n",
+    "# tf_checkpoint = TensorflowCheckpoint.from_directory(checkpoint_path)\n",
+    "\n",
+    "new_trainer = TensorflowTrainer(\n",
+    "    train_loop_per_worker=train_func,\n",
+    "    train_loop_config={\n",
+    "        \"batch_size\": 32,\n",
+    "        \"epochs\": 1,\n",
+    "    },\n",
+    "    scaling_config=ScalingConfig(\n",
+    "        num_workers=2,\n",
+    "        resources_per_worker={\"CPU\": 5},\n",
+    "        use_gpu=False,\n",
+    "    ),\n",
+    "    # NEW: Supply a checkpoint to resume from\n",
+    "    resume_from_checkpoint=tf_checkpoint\n",
+    ")\n",
+    "new_result = new_trainer.fit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "041bc1aa",
+   "metadata": {},
+   "source": [
+    "## Using the trained model for prediction\n",
+    "\n",
+    "The original Tensorflow quickstart tutorial doesn't actually go over loading the model for prediction, but let's see how you would complete the machine learning lifecycle with scalable batch prediction with Ray AIR!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f7fc01b",
+   "metadata": {},
+   "source": [
+    "### Loading the model for prediction\n",
+    "\n",
+    "Here's how to load the model from a checkpoint and perform model inference on some sample data.\n",
+    "\n",
+    "We retrieve the last reported checkpoint through the training {class}`Result <ray.air.result.Result>`, but the checkpoint can also be loaded from a checkpoint directory path via {meth}`TensorflowCheckpoint.from_directory <ray.air.checkpoint.Checkpoint.from_directory>`."
```

Comment:
we should show `result.checkpoint.to_directory()` or `result.checkpoint.to_uri()` for `from_directory` to make sense.

Otherwise I'm not sure where this directory comes from.

Reply:
I was thinking if a user wanted to load a checkpoint after they no longer have access to `result` (ex: if their script has finished running).

I might want to use this PR https://github.com/ray-project/ray/pull/29083 to get the results from an experiment path.